packages/contracts/contracts/identity/NonFungibleRegistryEnumerableUpgradeable.sol

```diff
@@ -273,7 +278,8 @@ contract NonFungibleRegistryEnumerableUpgradeable is
 
         _createLabeledToken(to, label, registrationData, tokenId);
 
-        uint256 burnAmount = registrationFee * burnFee / 10000;
+        uint256 burnAmount = calculateRoyalty(registrationFee);
```

Comment:
I don't think this is the intended functionality; this function costs "registration token" (ie Hypertoken) to use, and isn't/shouldn't be related to the royalties at all.